pkg/cmd/generate-bazel-extra/main.go

```diff
@@ -290,6 +290,30 @@ func generateTestsTimeouts() {
 		log.Fatal(err)
 	}
 	for size, timeout := range testSizeToDefaultTimeout {
+		if size == "enormous" {
```

Comment:
Please extract this part into a separate function `ExcludeSqliteLogicTests` and add a unit test since it's a pure function.